### ISSUE TITLE
No react native developer has a 12 years experience as of now.

### ISSUE THREAD

**Issue by hillkim** [ReactNativeGallery/reactnative-gallery-web#18]
This is not really an issue but I don't think there is a RN developer out there who has a 12 year experience. Not unless they were using it before  facebook created it lol! @xcarpentier . Am a junior one myself. But I wouldn't mind some gig if you have some.
![Screen Shot 2020-09-13 at 7 15 12 PM](https://user-images.githubusercontent.com/25987730/93023133-82e41600-f5f5-11ea-97ff-0d11068817e2.png)

**Comment by xcarpentier** [comment]
Yes, it’s confusing, I don’t have 12 years of experience in RN, you are right, but in general...
